packages/eslint-plugin/docs/rules/switch-exhaustiveness-check.mdx

```diff
@@ -78,6 +78,25 @@ switch (literal) {
 }
 ```
 
+### `defaultCaseCommentPattern`
+
+{/* insert option description */}
```

Comment:
Per @FloEdelmann's https://github.com/typescript-eslint/typescript-eslint/pull/10218#discussion_r1868991801, not being able to use `defaultOptions` means the default isn't being put into docs. So for now:

```suggestion
{/* insert option description */}

Default: `/^no default$/iu`.
```